_config.yml

```diff
@@ -38,7 +38,8 @@ email_url: oceanhkw@uw.edu
 # Build settings
 markdown: kramdown
 
-# Submodule setting
+# Theme setting, we are excluding jekyll-spectral-theme/ root to avoid picking up theme config.
+# We should probably clean it up later.
```

Comment:
The `exclude` here is to ignore some configs from the upstream submodule. We could just remove those file latest b/c they are related to the geohackweek.

Reply:
Sure. Removing "later" (I assume that's what you meant to say?) sounds good.